cli/src/test/kotlin/io/spine/protodata/cli/app/CliTest.kt

```diff
@@ -118,9 +127,10 @@ class `Command line application should` {
             "-p", DefaultOptionsCounterPlugin::class.jvmName,
             "-r", DefaultOptionsCounterRenderer::class.jvmName,
             "--src", srcRoot.toString(),
+            "-d", targetRoot.toString(),
```

Comment:
To emphasise the importance of this parameter, I would rather call it `--target`. `-d` also could mean "debug".